ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/AbstractCreateStreamCommand.java

```diff
@@ -196,6 +196,7 @@ private void validateWithClause(final Set<String> withClauseVariables) {
     validSet.add(DdlConfig.TOPIC_NAME_PROPERTY.toUpperCase());
     validSet.add(KsqlConstants.AVRO_SCHEMA_ID.toUpperCase());
     validSet.add(DdlConfig.TIMESTAMP_FORMAT_PROPERTY.toUpperCase());
+    validSet.add(DdlConfig.AVRO_SCHEMA_FULL_NAME.toUpperCase());
```

Comment:
Only valid if its a AVRO value format.  This function should throw if `AVRO_SCHEMA_FULL_NAME` is present, but the stream is not AVRO.

(Assuming you can determine if its AVRO at this point).

Ideally, add tests to cover both paths.

Reply:
Actually, I guess you're handling this else where - don't worry about it.